device/iot-device-client/src/main/java/com/microsoft/azure/sdk/iot/device/FileUpload.java

```diff
@@ -34,7 +33,7 @@
     FileUploadSasUriResponse getFileUploadSasUri(FileUploadSasUriRequest request) throws IOException
     {
         IotHubTransportMessage message = new IotHubTransportMessage(request.toJson());
-        message.setIotHubMethod(IotHubMethod.POST);
```

Comment:
IotHubMethod was a duplicate of the HttpsMethod class, so we can just delete it and use HttpsMethod everywhere instead